projects/plugins/jetpack/extensions/blocks/paywall/index.js

```diff
@@ -19,19 +19,27 @@ export const settings = {
 		foreground: getIconColor(),
 	},
 	category: 'earn',
-	keywords: [],
+	keywords: [
+		_x( 'email', 'block search term', 'jetpack' ),
+		_x( 'follow', 'block search term', 'jetpack' ),
+		_x( 'gated', 'block search term', 'jetpack' ),
+		_x( 'memberships', 'block search term', 'jetpack' ),
+		_x( 'newsletter', 'block search term', 'jetpack' ),
+		_x( 'signin', 'block search term', 'jetpack' ),
+		_x( 'subscribe', 'block search term', 'jetpack' ),
+		_x( 'subscription', 'block search term', 'jetpack' ),
+		_x( 'subscriptions', 'block search term', 'jetpack' ),
+	],
 	supports: {
 		customClassName: false,
 		html: false,
 		multiple: false,
 	},
+	parent: [ 'core/post-content' ],
```

Comment:
Took from https://github.com/WordPress/gutenberg/blob/4b4914b7e4ccdeb6393e95eee4666c5e62a92af3/packages/block-library/src/nextpage/block.json#L9

Reply:
perfect

Reply:
Note that this just limits to root of the document, but we still need to restrict to just post-editor and exclude custom post types.

This hook seems good next step: https://developer.wordpress.org/reference/hooks/allowed_block_types_all/